publisher/product/glossary/build.sh

```diff
@@ -115,390 +31,65 @@ function publishProductGlossaryRemoveWarnings() {
+  
+	if [ ${PIPESTATUS[0]} -ne 0 ] ; then
+		error "Could not collect FIBO ontologies"
+		return 1
+	fi
+  
+	logRule "Creating data dictionaries for DEV and PROD"
+  
+	#
+	# Set the memory for ARQ
+	#
+	JVM_ARGS="--add-opens java.base/java.lang=ALL-UNNAMED"
+	JVM_ARGS="${JVM_ARGS} -Dxxx=arq"
+	JVM_ARGS="${JVM_ARGS} -Xms2g"
+	JVM_ARGS="${JVM_ARGS} -Xmx4g"
+	JVM_ARGS="${JVM_ARGS} -Dfile.encoding=UTF-8"
+	JVM_ARGS="${JVM_ARGS} -Djava.io.tmpdir=\"${TMPDIR}\""
+	export JVM_ARGS
+	logVar JVM_ARGS
+	
+	${JENA_ARQ} --data ${TMPDIR}/dev.rdf --query ${SCRIPT_DIR}/product/glossary/data_dictionary.sparql --results=CSV > ${TMPDIR}/dev_proto_data_dictionary.csv
+	${JENA_ARQ} --data ${TMPDIR}/prod.rdf --query ${SCRIPT_DIR}/product/glossary/data_dictionary.sparql --results=CSV > ${TMPDIR}/prod_proto_data_dictionary.csv
+        
+	logRule "Extending data dictionaries with generated definitions"
+		
+	${PYTHON3} ${SCRIPT_DIR}/lib/dictionary_maker_no_sparql.py --input ${TMPDIR}/dev_proto_data_dictionary.csv --output ${glossary_product_tag_root}/glossary-dev.csv
+  	${PYTHON3} ${SCRIPT_DIR}/lib/dictionary_maker_no_sparql.py --input ${TMPDIR}/prod_proto_data_dictionary.csv --output ${glossary_product_tag_root}/glossary-prod.csv
+  
+	logRule "Writing from csv files to xlsx files"
 
-  OPTIONAL {?c owlnames:synonym ?syn}
-  OPTIONAL {?c owlnames:example ?example}
-  OPTIONAL {?c owlnames:explanatoryNote ?explanatoryNote}
+	touch "${glossary_product_tag_root}/glossary.log"	
 
-  OPTIONAL {?c  owlnames:mdDefinition ?GeneratedDefinition}
-}
-GROUP BY ?c ?Term ?Type ?Definition ?GeneratedDefinition ?example ?explanatoryNote ?ReleaseStatus
-ORDER BY ?Term
-__HERE__
+	${PYTHON3} ${SCRIPT_DIR}/lib/csv-to-xlsx.py \	
+	   "${glossary_product_tag_root}/glossary-prod.csv" \	
+	   "${glossary_product_tag_root}/glossary-prod.xlsx" \	
+	   "${glossary_script_dir}/csvconfig"	
 
-  ${JENA_ARQ} --data="${dataTurtle}" --query="${TMPDIR}/makeCcsv.sparql" --results=TSV > "${glossaryBaseName}-dev.tsv"
-  ${SED}  '/"Development"$/d' "${glossaryBaseName}-dev.tsv" > "${glossaryBaseName}-prod.tsv"
-  
-  ${SED} -i 's/"@../"/g; s/\t\t\t/\t""\t""\t/; s/\t\t/\t""\t/g; s/\t$/\t""/' "${glossaryBaseName}-dev.tsv"
-  ${SED} -i 's/"@../"/g; s/\t\t\t/\t""\t""\t/; s/\t\t/\t""\t/g; s/\t$/\t""/' "${glossaryBaseName}-prod.tsv"
-
-  ${SED} 's/"\t"/","/g; s/\\"/""/g' "${glossaryBaseName}-dev.tsv" > "${glossaryBaseName}-dev.csv"
-  ${SED} 's/"\t"/","/g; s/\\"/""/g' "${glossaryBaseName}-prod.tsv" > "${glossaryBaseName}-prod.csv"
-  ${SED} -i '1s/\t[?]/,/g;1s/^[?]//' "${glossaryBaseName}-dev.csv"
-  ${SED} -i '1s/\t[?]/,/g;1s/^[?]//' "${glossaryBaseName}-prod.csv"
 
-  touch "${glossary_product_tag_root}/glossary.log"
-  
-  ${PYTHON3} ${SCRIPT_DIR}/lib/csv-to-xlsx.py \
-    "${glossaryBaseName}-prod.csv" \
-    "${glossaryBaseName}-prod.xlsx" \
-    "${glossary_script_dir}/csvconfig"
+	${PYTHON3} ${SCRIPT_DIR}/lib/csv-to-xlsx.py \	
```

Comment:
remove spaces after "\" characters (also in following lines)

Reply:
This should be fixed by the patch.